www/src/components/marketplace/MarketplaceRepositories.js

```diff
@@ -61,16 +64,33 @@ function MarketplaceRepositories({ installed }) {
 
   const sortedRepositories = repositories.slice()
     .sort((a, b) => a.name.localeCompare(b.name))
-    .filter(repository => categories.length ? categories.includes(repository.category) : true)
+    .filter(repository => categories.length ? categories.includes(repository.category.toLowerCase()) : true)
```

Comment:
sorting should also be delegated to the api

Reply:
I've added a ticket for you to do it.